packages/alea-frontend/pages/study-buddy/[courseId].tsx

```diff
@@ -77,6 +86,8 @@ function StatsForModerator() {
   const [inactiveUsers, setInactiveUsers] = useState(0);
   const [numberOfConnections, setNumberOfConnections] = useState(0);
   const [unacceptedRequest, setUnacceptedRequest] = useState(0);
+  const [connections, setConnections] = useState<UserStats['connections']>([]);
```

Comment:
`UserStats['connections']`
I didn't know this was possible. is this a valid type?